app/controllers/admin/user_courses_controller.rb

```diff
@@ -0,0 +1,138 @@
+class Admin::UserCoursesController < AdminController
+  before_action :set_user_course, only: %i(approve reject)
```

Comment:
```suggestion
  before_action :set_user_course, only: %i(approve reject reject_detail)
```
Thêm action vào mới load đc chứ